mindjunkies/lecture/tests/test_views.py

```
@@ -97,28 +99,59 @@ def test_lecture_video_forbidden_for_non_enrolled(
+        "format": "video"
+    })
+
+    video_data = {
+        "title": "Test Lecture",
+        "video_url": "https://www.youtube.com/watch?v=dQw4w9WgXcQ"
+    }
+
+    response = client.post(url, video_data, follow=True)
+
+    assert response.status_code == 200
+    assert LectureVideo.objects.filter(lecture=lecture).exists()
```

Ruff (0.8.2)

Undefined name `LectureVideo`

(F821)